src/lib/error/api-error.test.ts

```diff
@@ -320,3 +322,14 @@ describe('OpenAPI error conversion', () => {
         expect(description.includes(illegalValue)).toBeTruthy();
     });
 });
+
+describe('Error serialization special cases', () => {
+    it('OwaspValidationErrors: adds `validationErrors` to `details`', () => {
+        const results = owasp.test('123');
+        const error = new OwaspValidationError(results);
+        const json = fromLegacyError(error).toJSON();
+
+        expect(json.details!![0].message).toBe(results.errors[0]);
```

Comment:
what's the double exclamation mark here?

Reply:
But maybe a single one'll do? I saw we did the double somewhere else, so I copied that.

Reply:
Update: yeah, a single one is enough.

Reply:
yeah I was surprised by a double exclamation mark :)